dipy/tracking/tests/test_stopping_criterion.py

```diff
@@ -41,11 +41,13 @@ def test_binary_tissue_classifier():
             state_boolean = btc_boolean.check_point(pts)
             state_float64 = btc_float64.check_point(pts)
             if mask[ind] > 0:
-                npt.assert_equal(state_boolean, TissueTypes.TRACKPOINT)
-                npt.assert_equal(state_float64, TissueTypes.TRACKPOINT)
+                npt.assert_equal(state_boolean,
+                                 int(StreamlineStatus.TRACKPOINT))
+                npt.assert_equal(state_float64,
+                                 int(StreamlineStatus.TRACKPOINT))
             else:
-                npt.assert_equal(state_boolean, TissueTypes.ENDPOINT)
-                npt.assert_equal(state_float64, TissueTypes.ENDPOINT)
+                npt.assert_equal(state_boolean, int(StreamlineStatus.ENDPOINT))
+                npt.assert_equal(state_float64, int(StreamlineStatus.ENDPOINT))
```

Comment:
@gabknight why this needs casting now?

Reply:
I don't know why, but on a single build of Travis, with python 3.5, the test was failing because it was comparing a ``int`` with a ``StreamlineStatus`` Enum type.  Somehow, with that build setup, assert_equal wasn't comparing the int value of the Enum but the actual type ``Enum <StreamlineStatus.ENDPOINT>`` with the ``int`` value on the other side.

Since all other tests passed and all other build worked fine, I just fixed the issue this way. Should I add a comment here?

Reply:
This was the failing build: Python: 3.5
DEPENDS="cython==0.29 numpy==1.9.0 scipy==1.0 nibabel==2.4.0 h5py==2.4.0"

Maybe the numpy version it too old here to correctly handle this.

Reply:
Makes sense. Thanks